runelite-client/src/main/java/net/runelite/client/plugins/timers/GameTimer.java

```diff
@@ -67,8 +67,8 @@
 	CHARGE(SpriteID.SPELL_CHARGE, GameTimerImageType.SPRITE, "Charge", 6, ChronoUnit.MINUTES),
 	STAFF_OF_THE_DEAD(ItemID.STAFF_OF_THE_DEAD, GameTimerImageType.ITEM, "Staff of the Dead", 1, ChronoUnit.MINUTES),
 	ABYSSAL_SIRE_STUN(ItemID.ABYSSAL_ORPHAN, GameTimerImageType.ITEM, "Abyssal Sire Stun", 30, ChronoUnit.SECONDS, true),
-	HOME_TELEPORT(SpriteID.SPELL_LUMBRIDGE_HOME_TELEPORT, GameTimerImageType.SPRITE, "Home Teleport", 30, ChronoUnit.MINUTES),
-	MINIGAME_TELEPORT(SpriteID.TAB_QUESTS_RED_MINIGAMES, GameTimerImageType.SPRITE, "Minigame Teleport", 20, ChronoUnit.MINUTES),
+	HOME_TELEPORT(SpriteID.SPELL_LUMBRIDGE_HOME_TELEPORT, GameTimerImageType.SPRITE, "Home Teleport", GraphicID.BOOK_HOME_TELEPORT_4, 30, ChronoUnit.MINUTES, false),
```

Comment:
I haven't checked with dev tools lately but I'm pretty sure the twisted themed book teleport and cow teleport both show different graphics than the standard home teleport.

Reply:
I tested the cow teleport and you're right, it does. I bet the twisted themed does as well, but I'm unable to test for it and figure out its graphic id right now. Is there another way to get the graphic id? I can't find it in the api.